src/lib/Components/Consignments/Screens/Overview.tsx

```diff
@@ -56,18 +55,31 @@ export default class Info extends React.Component<Props, ConsignmentSetup> {
   goToLocationTapped = () =>
     this.props.navigator.push({ component: Location, passProps: { updateWithResult: this.updateLocation } })
 
-  updateMetadata = (result: ConsignmentMetadata) => this.setState({ metadata: result })
-  updateProvenance = (result: string) => this.setState({ provenance: result })
+  updateArtist = (result: SearchResult) => this.updateStateAndMetaphysics({ artist: result })
+  updateMetadata = (result: ConsignmentMetadata) => this.updateStateAndMetaphysics({ metadata: result })
+  updateProvenance = (result: string) => this.updateStateAndMetaphysics({ provenance: result })
   updateLocation = (city: string, state: string, country: string) =>
-    this.setState({ location: { city, state, country } })
+    this.updateStateAndMetaphysics({ location: { city, state, country } })
```

Comment:
Indentation?

Reply:
nah, indentation is perfect ( I couldn't change it from this if I wanted to ) - if the one line goes past 120 chars, then it splits first here

Reply:
Ah, I see. Misunderstood line 61.